src/pages/about/governance.md

```diff
@@ -1,44 +1,51 @@
----
-id: governance
----
-
 # Tauri Governance
 
-## Sustainability
+One of the main goals of the organizational structure of Tauri is to guarantee the sustainability, health and well-being of its contributors. [The Tauri Programme within the Commons Conservancy](https://dracc.commonsconservancy.org/0035/) was established to facilitate an open, transparent and efficient governance process throughout the future development of the Tauri and its auxiliary materials.
 
-One of the main goals of the organizational structure of Tauri is to guarantee the sustainability of Tauri and the health and well-being of its contributors. The world of Open Source is fraught with peril and discord, and we have taken measures to ensure the longevity of Tauri. This document explains how we go about doing so.
+## Tauri Working Group
 
-## Organizational Structure
+The Tauri Working Group is the collective framework created to enable this governance process. It's composed of the following components:
 
-Tauri apps is governed by the community and work is done in the context of public working groups. Each working group has a dedicated channel on the Discord server as well as a Team on GitHub. Other than that, each WG is free to use whatever type of organizational model it chooses.
+- Working Group Members
+- Tauri Board & Board Directors
+- Domains & Domain Leads
+- Teams
 
-The current working groups are:
+![Tauri governance diagram](./governance-diagram.svg)
 
-- WG Governance & Guidance
-- WG Tech
-- WG Education
-- WG Media
-- WG Security
-- WG Devops
 
-With the exception of the security working group, which is by invite only and convenes privately, all other working groups are public and open to any and all participants.
+### Working Group Members
 
-Visit the [Governance and Guideance repository](https://github.com/tauri-apps/governance-and-guidance) to get more information.
+All of the individuals that make up the Tauri Working Group.
 
-## Code of Conduct
+### Tauri Board & Board Directors
 
-Everyone participating in the Tauri community is expected to follow the [Code of Conduct](https://github.com/tauri-apps/governance-and-guidance/blob/main/CODE_OF_CONDUCT.md).
+The collective Board is the central decision making body for the Tauri Programme. Although the Tauri Board is ultimately responsible for decisions, they may delegate to Domain Leads as they see fit.
```

Comment:
Who they might delegate to isn't restricted to Domain Leads. For example, Domain Leads may in turn delegate to Teams. So for this document I would describe it in terms of what they *typically* are concerned with.
- Major decisions (potentially impacting the future of Tauri)
- Concerned with overall project goals

Reply:
Good point on this, do you have some specific phrasing in mind to use?